projects/fal/src/fal/toolkit/file/providers/fal.py

```diff
@@ -91,11 +92,11 @@ def _refresh_token(self) -> None:
 
 @dataclass
 class ObjectLifecyclePreference:
-    expiration_duration_seconds: int
+    expiration_duration_seconds: int | None = None
 
 
 GLOBAL_LIFECYCLE_PREFERENCE = ObjectLifecyclePreference(
-    expiration_duration_seconds=86400
+    expiration_duration_seconds=None,
```

Comment:
Let's keep the old default. We talked about defaulting to never expire on the CDN side. We don't want to modify the SDK behavior

Reply:
reverted